pages/insight_menu.php

```diff
@@ -0,0 +1,67 @@
+
+<style>
+
+ul {
+    list-style: none;
+}
+
+* {
+    font-family: Arial;
+}
+
+#categoryPrice {
+    float:right;
+    font-weight: normal;
+    font-size:17px;
+}
+
+#price {
+    float:right;
+}
+
+</style>
+
+<?php
+
+function echoCategory($categoryId, $size = 1) {
+    $subcategories = base_query("SELECT * FROM DishCategory WHERE ParentCategoryId = :categoryId ORDER BY Position", [':categoryId' => $categoryId])->fetchAll();
+    $category = base_query("SELECT * FROM DishCategory WHERE Id = :categoryId", [':categoryId' => $categoryId])->fetch();
+?>
+<!-- Echo category name -->
+    <h<?= $size ?>>
+        <?= $category['Name']?>
+        <!-- Checks if a category has a price attached to itself -->
```

Comment:
Deze comment zal ge-echo'ed worden. Het is mooier om deze in de php tag te stoppen die je de volgende regel begint.